src/Disks/IO/AsynchronousReadIndirectBufferFromRemoteFS.cpp

```diff
@@ -170,71 +171,48 @@ bool AsynchronousReadIndirectBufferFromRemoteFS::nextImpl()
     if (!hasPendingDataToRead())
         return false;
 
-    Stopwatch watch;
-    CurrentMetrics::Increment metric_increment{CurrentMetrics::AsynchronousReadWait};
-
-    size_t size = 0;
-    size_t bytes_read = 0;
+    size_t size, offset;
```

Comment:
These variables are initialised two lines below (in both `if` and `else`) and nothing is done before they are initialised, so I'd rather not to do it here.

Reply:
Variable initialization is more about saving some developer time in future which can never happened :) In case of a variable is not initialized as expected (due to some local changes), with default initialization it'd lead to incorrect but stable behavior, w/o initialization it'll be incorrect and unstable. Other than that, everything is ok.